templates/shared/_cloud-survey-banner.html

```diff
@@ -2,7 +2,9 @@
   <div class="{% if cols %}col-{{ cols }}{% else %}col-12{% endif %}">
     <div class="p-notification">
       <div class="p-notification__content">
-        <p class="p-notification__message">Fill in the <a href="https://docs.google.com/forms/d/e/1FAIpQLSdDECWbUMkJ1gqk8iOylEF9sli23tuMWHQRVrC2JUOzSDM_Tg/viewform">Cloud Pricing Survey 2022</a> and have a chance to win a virtual pass to KubeCon Europe 2022!</p>
+        <p class="p-notification__message">
+          <a href="https://docs.google.com/forms/d/e/1FAIpQLSdDECWbUMkJ1gqk8iOylEF9sli23tuMWHQRVrC2JUOzSDM_Tg/viewform">Win a virtual pass to KubeCon Europe 2022!</a>
```

Comment:
```suggestion
          <a href="https://docs.google.com/forms/d/e/1FAIpQLSdDECWbUMkJ1gqk8iOylEF9sli23tuMWHQRVrC2JUOzSDM_Tg/viewform" target="_blank">Win a virtual pass to KubeCon Europe 2022!</a>
```
To have it open in a new tab so the user doesn't lose where they were on the site

Reply:
I don't know if we have a team policy on this, I think we might have discussed it a dev catchup, but IMO the user is able to make the decision on whether they want to open a link in a new tab using middle click/ctrl+click, we shouldn't make that decision for them.